src/fetch/payouts.py

```diff
@@ -308,6 +312,14 @@ def prepare_transfers(payout_df: DataFrame, period: AccountingPeriod) -> PeriodP
             overdrafts.append(overdraft)
         transfers += payout_datum.as_payouts()
 
+    transfers.append(
+        Transfer(
+            token=None,
+            recipient=PROTOCOL_FEE_SAFE,
+            amount_wei=payout_df.protocol_fee_eth.sum(),
```

Comment:
I believe i had cast the right-hand side to `int`. Will need to double-check if this is still needed.

Reply:
I added the cast. (It is a bit worrying that we do not have a test to check for that.)